### PR TITLE
Add cookbook for working with Parquet

### PR DESCRIPTION
**Pull request by harterrt**
You can preview the resulting documentation here:
https://harterrt.github.io/firefox-data-docs/cookbooks/parquet.html

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/firefox-data-docs/11)

### CODE REVIEW

**Comment by harterrt** [comment]
@vitillo, any comments on the ["Where to Save Data" section](https://harterrt.github.io/firefox-data-docs/cookbooks/parquet.html#where-to-save-data)?

**Comment by trink** [comment]
It should have a direct to parquet section but you can file an issue against myself or @fbertsch

**Comment by harterrt** [comment]
Thanks, @trink! Filed [Issue 12](https://github.com/mozilla/firefox-data-docs/issues/12)

File: cookbooks/parquet.md [hunk 1]
```diff
@@ -1,4 +1,61 @@
...
+
+# Writing Parquet Tables
+
+Saving a table to parquet is a great way to share an intermediate dataset.
+
+## Where to save data
+
+We have a test bucket used for temporary storage while prototyping analyses.
+Feel free to create a subdirectory in this bucket: `s3://telemetry-test-bucket/`.
+**Data in the test bucket may be deleted**.
```
Comment (vitillo) [inline cookbooks/parquet.md]: Every user has it's own "directory" on S3, which is where they should be writing datasets to; @mreid-moz can comment on it. The test bucket is used internally by the data platform team.

File: cookbooks/parquet.md [hunk 2]
```diff
@@ -1,4 +1,61 @@
...
+
+When your analysis is production-ready,
+save it to a subdirectory of `s3://telemetry-parquet/`
```
Comment (vitillo) [inline cookbooks/parquet.md]: That bucket is used by the data platform team and we don't want outside users to write data in there.

File: cookbooks/parquet.md [hunk 3]
```diff
@@ -1,4 +1,61 @@
...
+# Working with Parquet
 
-Work being tracked 
-[in this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1358210).
+This guide will give you a quick introduction to working with 
+[Parquet](https://parquet.apache.org/) files at Mozilla.
+You can also refer to Spark's documentation on the subject
+[here](http://spark.apache.org/docs/latest/sql-programming-guide.html#parquet-files).
+
+Most of our [derived datasets](/datasets/derived.md),
```
Comment (vitillo) [inline cookbooks/parquet.md]: There should be a mention of direct-to-parquet here as well.

File: cookbooks/parquet.md [hunk 4]
```diff
@@ -1,4 +1,61 @@
...
+save it to a subdirectory of `s3://telemetry-parquet/`
+
+## How to save data
+
+You can save the dataframe `test_dataframe`
+ to the `telemetry-test-bucket` with the following command:
+
+```python
+test_dataframe.write.mode('error') \
+    .parquet('s3://telemetry-test-bucket/my_subdir/table_name.parquet')
```
Comment (vitillo) [inline cookbooks/parquet.md]: The table name shouldn't have a `.parquet` suffix.

File: cookbooks/parquet.md (hunk 3 above)
Comment (harterrt) [inline cookbooks/parquet.md]: https://github.com/mozilla/firefox-data-docs/issues/12

File: cookbooks/parquet.md (hunk 2 above)
Comment (harterrt) [inline cookbooks/parquet.md]: I'll change this to the following, SG?

When your analysis is production ready, open a PR against [python_mozetl](https://github.com/mozilla/python_mozetl).

File: cookbooks/parquet.md (hunk 1 above)
Comment (harterrt) [inline cookbooks/parquet.md]: OK, so no user outside of data platform should be writing to telemetry-test-bucket. I'll look out for notes on the user specific directory from mreid.

Comment (mreid-moz) [inline cookbooks/parquet.md]: The recommended location for persistent analysis outputs is `s3://net-mozaws-prod-us-west-2-pipeline-analysis/<username>/`, so for example I have some stuff in `s3://net-mozaws-prod-us-west-2-pipeline-analysis/mreid/`.

This bucket is readable and writable from clusters launched via ATMO or Airflow.

File: cookbooks/parquet.md (hunk 2 above)
Comment (vitillo) [inline cookbooks/parquet.md]: Sounds good.

File: cookbooks/parquet.md (hunk 1 above)
Comment (mreid-moz) [inline cookbooks/parquet.md]: Note that we want to abstract the actual S3 location out per [Bug 1360630](https://bugzilla.mozilla.org/show_bug.cgi?id=1360630)

Comment (harterrt) [inline cookbooks/parquet.md]: Fixed

File: cookbooks/parquet.md (hunk 2 above)
Comment (harterrt) [inline cookbooks/parquet.md]: Fixed

File: cookbooks/parquet.md (hunk 4 above)
Comment (harterrt) [inline cookbooks/parquet.md]: Fixed

**Comment by harterrt** [comment]
Thanks, @vitillo, RFAL.

**Pull request merged**